queue.go

```diff
@@ -13,11 +14,14 @@ const (
 	purgeBatchSize      = int64(100)
 )
 
+var randSrc = rand.New(rand.NewSource(time.Now().UnixNano()))
+
 type Queue interface {
 	Publish(payload ...string) error
 	PublishBytes(payload ...[]byte) error
 	SetPushQueue(pushQueue Queue)
 	StartConsuming(prefetchLimit int64, pollDuration time.Duration) error
+	StartConsumingWithJitter(prefetchLimit int64, pollDuration time.Duration) error
```

Comment:
Instead of introducing a completely new method why not to just update StartConsuming with an extra bool jitter parameter?